### ISSUE TITLE
Support for Carbon 3

### ISSUE THREAD

**Issue by holtkamp** [Webklex/php-imap#483]
**Is your feature request related to a problem? Please describe.**
No, more a "nice to have"

**Describe the solution you'd like**
Allow Carbon 3

https://github.com/briannesbitt/Carbon/releases

**Additional context**
Carbon 3 can be installed just fine. 

I tried [running the tests](https://github.com/Webklex/php-imap#testing), but it seems it requires a live mailbox which I do not have for testing:

```
This test requires a live mailbox. Please set the LIVE_MAILBOX environment variable to run this test.
```

**Comment by mhpcc** [comment]
I've forced it to use Carbon 3 for a while, since 2.x does not support Symfony 7+.

You can add `"nesbot/carbon": "3.x-dev as 2.99",``to your composer.json and it'll work.
But yeah, great that won't be necessary soon. Thank you webklex :)

**Comment by curry684** [comment]
Can the updated requirements getting a tag? Need this in a Symfony 7 production application and not really liking the alias   😉

**Comment by salimkanoun** [comment]
Yes it would be great if we can get a release as there is also a conflict of carbon in Laravel 11 apps.

Thank you all maintainers of this library, this work is much appreciated

**Comment by viktorolivares** [comment]
Sería bueno agregar la compatibilidad de Carbon 3 en el package para Laravel 11

**Comment by Webklex** [comment]
Hi @holtkamp @mhpcc @curry684 @salimkanoun @viktorolivares @MrJuliuss ,
thanks for your patience <3

A new release is available, with more changes coming :)

Best regards & happy coding,
